care/facility/models/mixins/permissions/facility.py

```diff
@@ -65,11 +66,7 @@ def has_object_read_permission(self, request):
         )
 
     def has_object_write_permission(self, request):
-        if (
-            request.user.user_type == User.TYPE_VALUE_MAP["DistrictReadOnlyAdmin"]
-            or request.user.user_type == User.TYPE_VALUE_MAP["StateReadOnlyAdmin"]
-            or request.user.user_type == User.TYPE_VALUE_MAP["StaffReadOnly"]
-        ):
+        if request.user.user_type in READ_ONLY_USER_TYPES:
             return False
         if request.user.user_type < User.TYPE_VALUE_MAP["Staff"]:  # todo Temporary
```

Comment:
@vigneshhari can we remove this temporary bypass?
This mixin is only used for facility object